openshift/e2e-common.sh

```diff
@@ -179,13 +180,18 @@ function install_catalogsource(){
   # And checkout the setup script based on that commit.
   local SERVERLESS_DIR=$(mktemp -d)
   local CURRENT_DIR=$(pwd)
-  git clone --depth 1 https://github.com/openshift-knative/serverless-operator.git ${SERVERLESS_DIR}
+  git clone -b support-passthrough --depth 1 https://github.com/nak3/serverless-operator.git ${SERVERLESS_DIR}
```

Comment:
We will have to revert this I guess when S-O work is done?

Reply:
Yes, that's correct.
This PR has a lot of manual change so I will clean up it after 
https://github.com/openshift/knative-serving/pull/1183 and S-O work were done.

Reply:
Just in case, I think we don't need to merge this (and you don't need to review this) until S-O was merged.
This PR just demonstrates that the backport with enabling option works fine.

Reply:
Yeah ok